cmd/runc-wrapper/runc-wrapper.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+const (
+	runc        string = "/usr/libexec/docker/docker-runc-current"
+	logLocation string = "/tmp/nspawn-runc-wrapper.log"
+)
+
+func main() {
+	var newArgs []string
+
+	fd, err := os.OpenFile(logLocation, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0660)
+	if err != nil {
+		log.Println("error opening logs, skipping:", err)
+	}
+	log.SetOutput(fd)
+	defer fd.Close()
+
+	log.Printf("old args: %+v", os.Args[1:])
+
+	for _, a := range os.Args[1:] {
+		if a == "" {
+			a = "\"\""
```

Comment:
I tried it without that before, but Go also parses it as an empty argument and I need to explicitly tell it to write that into the new arguments...

First is before removing that block, second after:
```
2017/05/18 12:11:35 old args: [--console  --blabla sadlfkjldkfjlaksdfj]
2017/05/18 12:11:35 new args: [--console "" --blabla sadlfkjldkfjlaksdfj]
---
2017/05/18 14:54:07 old args: [--console  --blabla sadlfkjldkfjlaksdfj]
2017/05/18 14:54:07 new args: [--console  --blabla sadlfkjldkfjlaksdfj]
```